submissions/DevBoard.md

```diff
@@ -0,0 +1,70 @@
+# Hackathon Submission: DevBoard
+
+
+## GitHub handles of Team Members  
+_Enter the GitHub handles of all your team members (including yourself) as separate bullet points_
+
+- @Aniket762
+- @Nikhil-Namboodiri
+- @Aniruddha-Raje
+- @Aayush-Garg-18
+
+## Project Title
+_What is the name of your project?_
+
+DevBoard
+
+## Project Description    
+_What have you built during the hackathon?_
+
+DevBoard is a comprehensive project management tool designed to streamline collaboration and enhance productivity for development teams. It integrates task management, real-time communication, and progress tracking in a single, intuitive platform.
+
+## Inspiration behind the Project  
+_What is the story behind this project? Why did you choose to work on this specific idea?_
+
+As developers, we often found ourselves juggling multiple tools for project management, communication, and task tracking. We wanted to create a unified solution that would simplify the development process and foster better team collaboration. DevBoard was born out of this need for a more efficient and integrated approach to project management in the software development world.
+
+## Tech Stack    
+_How have you built this project? Mention the technologies/methods/platforms you used to build your project._
+
+- Frontend: React.js with TypeScript
+- Backend: Node.js with Express
+- Database: MongoDB
+- Real-time Communication: Socket.io
+- Authentication: JWT (JSON Web Tokens)
+- Styling: Tailwind CSS
+- Deployment: Docker and Kubernetes
+- Version Control: Git and GitHub
+
+### Usage of Appwrite
+_Highlight any Appwrite products used within the project._
+
+- Appwrite Databases
+
+We used Appwrite Databases to store and manage project data, user profiles, and task information. The flexible schema and real-time capabilities of Appwrite Databases allowed us to create a responsive and dynamic user experience.
+
+- Appwrite Authentication
+
+Appwrite Authentication was implemented to handle user registration, login, and session management securely. This ensured that only authorized team members could access project-specific information.
+
+- Appwrite Storage
+
+We utilized Appwrite Storage for file management within projects. This allowed team members to easily upload, store, and share project-related documents and assets.
+
+- Appwrite Functions
+
+Appwrite Functions were used to implement custom backend logic, such as automated notifications and task assignments, enhancing the overall functionality of DevBoard.
+
+## Project Repo  
+_Share a public repo link of your project_
+
+https://github.com/Discord-Group-Project/DevBoard
```

Comment:
Please make the GitHub repo of the project public @Himanshuch8055

Reply:
> Please make the GitHub repo of the project public @Himanshuch8055 

Sorry, I will make it public. Give me some time!

Reply:
No problem! Make sure to make it public before Oct 31st and re-request a PR review  from me once it is ready